### ISSUE TITLE
[BUG] DataWeave valid Type annotation is marked as unable to resolve, but parsed correctly

### ISSUE THREAD

**Issue by KaiGrassnick** [mulesoft-labs/data-weave-intellij-plugin#93]
I'm getting this issue while writing dwl functions.

![image](https://user-images.githubusercontent.com/7880861/211250128-9197dafd-21c0-4272-a090-555e814d4849.png)

it also happens with other types. They are totally valid and the Plugin also parses it correctly ( gives a different error if the object is not an object, e.g. an array and marks the content ).

What should happen? Plugin should not mark the default types as unable to resolve

**Comment by machaval** [comment]
Yes, this happens when DW doesn't found wlang library in the classpath. Make sure you have it as a dependency
